kahuna/public/js/preview/image.js

```diff
@@ -111,7 +111,7 @@ image.directive('uiPreviewImageLarge', ['observe$', 'inject$', 'imgops',
 
                 const optimisedImage$ = image$.flatMap((image) => {
                     return Rx.Observable.fromPromise(imgops.getFullScreenUri(image));
-                });
+                }).debounce(5);
```

Comment:
According to [the docs](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/debounce.md) it is, buuuut I tried higher and it drops too many. Even at 10 it won't do they next image along. Any ideas?